app/partials/app.cjsx

```diff
@@ -11,7 +11,8 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
```

Comment:
Right. This seemed to be the best course for now since we are restricting logging to Gravity Spy. We will likely need to change this in the future as we expand the projects that contain logging from geordi.